src/pages/index.jsx

```diff
@@ -8,7 +8,7 @@ export default function Home() {
     <div>
       <PageTitle title="Home" />
       <Header />
```

Comment:
This is where Jared's `Layout` should go.

Reply:
Agreed, we just haven't added that in yet but it's coming in the next commit! :)